examples/http-caching-client/http_caching_client.client.out

```diff
@@ -1,5 +1,5 @@
 # Invoke the service.
```

Comment:
This line is redundant as we have added `Invoke the service as follows.` in the MD file before placing client.out files
Shall we fix them too?

Reply:
Yeah sure.

Reply:
Fixed with [cddb02c200585b2c72d855a9b47b485ad2e6d0b5](https://github.com/ballerina-platform/ballerina-distribution/pull/3024/commits/cddb02c200585b2c72d855a9b47b485ad2e6d0b5)